### ISSUE TITLE
getifaddr raise error

### ISSUE THREAD

**Issue by perfecto25** [arcage/net_sample.cr#5]
trying to get list of ifaces, getting raise
```
require "net_sample"

puts NetSample::NIC["lo"]
```

```
In lib/net_sample/src/net_sample/nic/linux.cr:36:19

 36 | raise Errno.new("errno #{Errno.value} on getifaddr()")

```

this line is raising on LibC

```
    if LibC.getifaddrs(out ifaddrs) == -1
      raise Errno.new("errno #{Errno.value} on getifaddr()")
    end
```

crystal 0.35.1
Mint 19.1 (x64)

**Comment by arcage** [comment]
Thank you for reporting.

I will check this.

**Comment by arcage** [comment]
@perfecto25 

I had released ~v0.2.3~ v0.2.4 to solve this.

Please check it.

Thank you again.

**Comment by arcage** [comment]
I will close this issue once.
